test/unit/specs/core/network/sendBeaconFactory.spec.js

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import sendBeaconFactory from "../../../../../src/core/network/sendBeaconFactory";
+
+// When running these tests in IE 11, they fail because IE doesn't like the
+// way the blob is constructed (see
+// https://github.com/bpampuch/pdfmake/pull/297/files for a workaround).
+// Fortunately, if navigator.sendBeacon doesn't exist (IE 11), sendBeaconFactory
+// should never be used (see createNetworkStrategy.js), so we can skip
+// these tests altogether.
+if (window.navigator.sendBeacon) {
+  describe("sendBeaconFactory", () => {
+    it("falls back to fetch if sendBeacon fails", () => {
+      const navigator = {
```

Comment:
I think it would be better to mark the test as pending so we can see which tests were not run for ie in the reports.

something like this:
```
describe("sendBeaconFactory", () => {
  it("falls back to fetch if sendBeacon fails", () => {
    if (!window.navigator.sendBeacon) {
      pending("No sendBeacon API available");
      return;
    }
   const navigator = {
   ...
```

Reply:
K, I'll make the change.